openupgrade_scripts/apriori.py

```diff
@@ -44,6 +44,20 @@
     # OCA/server-tools
     "openupgrade_records": "upgrade_analysis",
     # OCA/...
+    
+    # Tvtmaaddons
```

Comment:
@hieu1211 chú sửa như này nhé
```suggestion
    # Viindoo/tvtmaaddons
```